lib/workers/repository/update/branch/index.ts

```diff
@@ -278,7 +308,6 @@ export async function processBranch(
         'Branch + PR exists but is not scheduled -- will update if necessary'
       );
     }
-    await checkoutBranch(config.baseBranch!);
```

Comment:
?

Reply:
Moved it after the `canSkipBranchUpdateCheck` so we don't clone ifthe updates are same.